src/AntiCSRF.php

```diff
@@ -94,17 +94,17 @@ public static function getTokenArray($lockto = null)
         }
 
         if (empty($lockto)) {
-            $lockto = isset($_SERVER['REQUEST_URI']) 
+            $lockto = isset($_SERVER['REQUEST_URI'])
                 ? $_SERVER['REQUEST_URI']
                 : '/';
         }
 
-        if (\preg_match('#/$#', $lockto)) {
```

Comment:
Thanks, and my condolences for having to support an unsupported version of PHP. Best of luck.